pkgs/development/python-modules/tensorstore/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  numpy,
+  setuptools-scm,
+}:
+buildPythonPackage rec {
+  buildInputs = [setuptools-scm];
+  checkInputs = [pytestCheckHook];
+  meta = with lib; {
```

Comment:
usually `meta` is at the bottom of the file ( I haven't seen a case when it isn't )